helm/rook-operator/charts/ceph-cluster/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if ne .Values.global.cephClusterNamespace .Values.operatorNamespace }}
+---
+# Allow the ceph mgr to access cluster-wide resources necessary for the mgr modules
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-mgr-cluster-{{ .Values.global.cephClusterNamespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: rook-ceph-mgr-cluster
+subjects:
+  - kind: ServiceAccount
+    name: rook-ceph-mgr
+    namespace: {{ .Values.global.cephClusterNamespace }}
```

Comment:
we cannot use `.Release.Namespace` here (as upstream does) as this chart will be installed in `giantswarm` along with the operator chart.